lib/policies/event_hub_policy.js

```diff
@@ -1,6 +1,2 @@
 'use strict';
-
-var u = require('../utilities'),
-    ServiceBusPolicy = require('./service_bus_policy');
-
-module.exports = u.deepMerge({}, ServiceBusPolicy);
+module.exports = require('./service_bus_policy');
```

Comment:
Should have done this a long time ago. I kept expecting EH to need some actual overrides :)